Packs/Confluera/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Confluera",
+    "description": "Confluera-Cortex XSOAR integration pack",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "confluera",
+    "url": "https://www.confluera.com",
+    "email": "",
```

Comment:
Better to provide URL or email address of your support to which users can reach out in case problems come up with the pack